packages/esm-patient-chart-app/src/clinical-views/encounter-tile/encounter-tile.component.tsx

```diff
@@ -65,13 +68,13 @@ const EncounterData: React.FC<{
       <span className={styles.tileTitle}>{t(column.header)}</span>
       {!(obsValue === '--' && summaryValue !== '--' && !isNil(summaryValue)) && (
         <span className={styles.tileValue}>
-          <p>{obsValue}</p>
+          <p>{withUnit(obsValue, units)}</p>
         </span>
       )}
 
       {!isNil(summaryValue) && summaryValue !== '--' && (
         <span className={styles.tileValue}>
-          <p>{summaryValue}</p>
+          <p>{withUnit(summaryValue, units)}</p>
```

Comment:
Better to leave the units out of the summary value because most times, its used to show the date of the encounter etc